addon/middleware/index.js

```diff
@@ -0,0 +1,5 @@
+import thunk from 'redux-thunk';
```

Comment:
Whoa!! This just works out of the box?! I was under the impression redux-thunk refused to be imported in `addon`. But if it works, then great (it also means I've lied in my PR to the thunk shim repo https://github.com/ember-redux/ember-redux-thunk-shim/pull/19)